TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -28,7 +29,13 @@ public class Scanner
         /// <summary>
         /// Scan a line of a document
         /// </summary>
-        public static void ScanTokensLine(TokensLine tokensLine, MultilineScanState initialScanState, TypeCobolOptions compilerOptions, List<RemarksDirective.TextNameVariation> copyTextNameVariations)
+        /// <param name="tokensLine"></param>
+        /// <param name="initialScanState"></param>
+        /// <param name="compilerOptions"></param>
+        /// <param name="copyTextNameVariations"></param>
+        /// <param name="multiStringConcatBitPosition">Bit array of Multi String concatenation positions</param>
+        public static void ScanTokensLine(TokensLine tokensLine, MultilineScanState initialScanState, TypeCobolOptions compilerOptions, List<RemarksDirective.TextNameVariation> copyTextNameVariations,
+            BitArray multiStringConcatBitPosition = null)
```

Comment:
I'm a bit cautious about parameters with default values. It works well for option-like parameters meaning that the method works as well with or without the argument but here I don't know. Theoretically speaking we should always have this position array to scan the line correctly.

Reply:
Why not use the scan state instead to store this data ?

Reply:
As we already have a big memory consumption and scan state is instancied on each line, I think it's better to pass this information as a parameter.